### DISCUSSION TITLE
part-ii/2021-125/

### DISCUSSION THREAD

**Discussion by tripos-comments** [tripos-education/maths-tripos-questions#73] [Question comments]
Part II 2021 Quantum Information and Computation: Paper 1, Section I, $10 D$ 

https://questions.tripos.org/part-ii/2021-125/

**Comment by thenewtonium** [comment]
<details>
<summary><b>(b)</b> – idea is just that</summary>

If no-cloning didn't hold, then suppose Alice sends messages $\ket{a_0} = \ket{c_0}$, $\ket{a_1} = \ket{c_1}$ each with prob. 1/2 as in the stem of question.

Then since Bob can clone these (non-orthogonal) states, he can effectively measure Alice's message an arbitrary number of times, and so raise the probability of distinguishing them arbitrarily close to 1. Hence Bob can violate the Helstrom-Holevo bound (as $P_S<1$ for non-orthogonal states).

</details>
